test/functional/plugin/lsp/inlay_hint_spec.lua

```diff
@@ -60,13 +59,13 @@ describe('inlay hints', function()
             end,
           }
         })
-      ]], response)
+      ]],
+        response
+      )
     end)
 
-    it(
-      'inlay hints are applied when vim.lsp.inlay_hint(true) is called',
-      function()
-        local res = exec_lua([[
+    it('inlay hints are applied when vim.lsp.inlay_hint.toggle(true) is called', function()
```

Comment:
Idk if there are any guidelines regarding how to name `it` tests but I think they usually start with a verb and are phrased like: "applies inlay hints when toggle is called".

Reply:
sounds good, re-organized the test hierarchy in this file to fit that better